tests/JavaTest.java

```diff
@@ -1027,14 +1027,9 @@ public static void testBuilderGrowth() {
         TestEq("This is a small string", FlexBuffers.getRoot(b).asString());
 
         FlexBuffersBuilder failBuilder = new FlexBuffersBuilder(ByteBuffer.allocate(1));
-        try {
-            failBuilder.putString("This is a small string");
-            // This should never be reached, it should throw an exception
-            // since ByteBuffers do not grow
-            assert(false);
-        } catch (java.lang.ArrayIndexOutOfBoundsException exception) {
-            // It should throw exception
-        }
+        // Should never throw an exception
```

Comment:
This isn't currently true since there is the `throw new OutOfMemoryError()` in the call path.

Reply:
This is using the old `ByteBuffer` not the `ReadWriteBuf` interface. So this test shouldn't be affected by this change.

Reply:
Actually I don't get why This test was removed. Is this change allowing the Buffer to grow using `ByteBuffer`? @greenrobot-team

Reply:
That is a bug from my side. It should be: 
```java
public FlexBuffersBuilder(ByteBuffer bb, int flags) {
    this(new ByteBufferReadWriteBuf(bb), flags);
}
```
And it should fail to grow.
I will submit an MR later or if you feel inclined, you can add to this MR as well.

Reply:
Okay. I will take a look. Thanks!